src/components/Cards/MeetingCard.tsx

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+import styled from "@emotion/styled";
+import { Card } from "./";
+import colors from "../../styles/colors";
+import { fontSizes } from "../../styles/fonts";
+
+type MeetingCardProps = {
+  imgSrc: string;
```

Comment:
@JacksonMaxfield  Are we going to also add thumbnail_hover_file (animated gif of the meeting, when the user hovers over the card) to the meeting card?